pom.xml

```diff
@@ -66,6 +66,7 @@
         <sonar-maven-plugin.version>3.9.1.2184</sonar-maven-plugin.version>
         <spotless-maven-plugin.version>2.27.2</spotless-maven-plugin.version>
         <swagger-annotations.version>2.1.0</swagger-annotations.version>
+        <jjwt.version>0.11.5</jjwt.version>
```

Comment:
I would suggest to keep abc order for list of properties

Reply:
@snuyanzin better like this. updated.

Reply:
yes, great
thanks!